mitosheet/src/mito/components/elements/Dropdown.tsx

```diff
@@ -216,6 +221,18 @@ const Dropdown = (props: DropdownProps): JSX.Element => {
             return;
         }
 
+        // If the user clicks on the scroll up or scroll down buttons,
+        // then we don't want to close the dropdown
```

Comment:
I wonder if we might be forgetting to stop an event propagation here -- because clicking still sometimes closes the dropdown for me, at least when clicking the up arrow. See attached video. 


Uploading Scroll Bug.mov…